backend/app/services/channels/telegram/handler.py

```
@@ -0,0 +1,626 @@
+
+        if value == "cloud":
+            await device_selection_manager.set_cloud_executor(user.id)
+            return "✅ 已切换到**云端执行模式**"
+
+        if value == "device":
+            # Check if user has a device selected
+            selection = await device_selection_manager.get_selection(user.id)
+            if selection.device_id:
+                return "✅ 已切换到**设备模式**"
+            else:
+                return "❌ 尚未选择设备，请先使用 `/devices` 选择一个设备"
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)